dist/uPlot.esm.d.ts

```diff
@@ -445,7 +445,7 @@ declare namespace uPlot {
 		setSelect?:  ((self: uPlot) => void)[];
 
 		/** fires after a series is toggled or focused */
-		setSeries?:  ((self: uPlot, seriesKey: string, opts: Series) => void)[];
+		setSeries?:  ((self: uPlot, seriesIndex: number, opts: Series) => void)[];
```

Comment:
lets stay consistent and call it `seriesIdx` as elsewhere.

Reply:
Sorry! Didn't check naming conventions 😊